docs/gitlab.md

```diff
@@ -0,0 +1,88 @@
+
+#### Bot Server setup
+
+The server setup for Renovate Pro for GitLab is essentially the same as for GitHub Enterprise, so you can look at those examples. Essentially you need a low-mid sized server with Docker capabilities.
+
+## Configuration
+
+#### Renovate Pro environment variables
+
+Renovate Pro requires configuration via environment variables in addition to Renovate OSS's regular configuration:
+
+**`ACCEPT_AGREEMENT`**: Renovate Pro will not run unless you accept the terms of the [Renovate User Agreement](https://renovatebot.com/user-agreement) by setting this environment variable to `y`. This is required whether you are running Renovate Pro in commercial or evaluation mode.
+
+**`LICENSE_MODE`**: If you have purchased a commercial license for Renovate Pro then you need to set this value to `commercial` to enable more than 3 repositories and remove the evaluation mode banner from PRs. Leave this field empty to default to evaluation mode.
+
+**`LICENSE_NAME`**: To enable commercial mode, you also need to also fill in the company name that the license is registered to. It should match what you entered in the order form. Leave empty for evaluation mode.
+
+**`WEBHOOK_SECRET`**: This configuration option must be set unless you configured it to 'renovate', which is default.
+
+**`SCHEDULER_CRON`**: This configuration option accepts a 5-part cron schedule and is _optional_. It defaults to `0 * * * *` (i.e. once per hour exactly on the hour) if it is unset. If decreasing the interval then be careful that you do not exhaust the available hourly rate limit of the app on GitHub Enterprise or cause too much load.
+
+#### npm registry configuration
+
+If using your own internal npm registry, you may find it easiest to update your `docker-compose.yml` to include a volume that maps an `.npmrc` file to `/home/node/.npmrc`. The RC file should contain `registry=...` with the registry URL your company uses internally. This will allow Renovate to find shared configs and other internally published packages.
+
+#### Core Renovate Configuration
+
+The "core" Renovate functionality (i.e. same functionality you'd find in the CLI version or the hosted app) can be configured using environment variables (e.g. `RENOVATE_XXXXXX`) or via a `config.js` file that you mount inside the Renovate Pro container to `/usr/src/webapp/config.js`. Here are some essentials:
+
+**`RENOVATE_PLATFORM`**: Set this to "gitlab".
+
+**`GITLAB_ENDPOINT`**: This is the API endpoint for your GitLab host.
+
+**`GITLAB_TOKEN`**: A Personal Access Token for the GitLab bot account.
+
+**`GITHUB_TOKEN`**: A Personal Access Token for any user account on github.com. This is used for retrieving changelogs and release notes from repositories hosted on github.com and it does not matter who it belongs to. It needs only read-only access privileges. Note: do not configure `GITHUB_TOKEN`.
```

Comment:
This is confusing to me. It says `do not configure`, yet this part of the documentation is explaining why you would want to configure it.